app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -195,9 +197,9 @@ def process_templates(objects)
 
   def process_template(object)
     # Get the basic information:
-    uid = object.metadata.uid
-    name = object.metadata.name
-    domain = object.spec.template.spec.domain
+    metadata = object.metadata
+    uid = metadata.uid
+    name = metadata.name
 
     # Add the inventory object for the template:
     template_object = template_collection.find_or_build(uid)
```

Comment:
@agrare  Is there a place where we can store namespace?

Reply:
We don't have a container_project relationship on vms, we could either add one with a migration or model the project/namespace as either a folder or a label

Reply:
👍 works for me

Reply:
I will follow your suggestion in different PR.